h2o-py/tests/testdir_apis/H2O_Module/pyunit_h2ocluster_shutdown.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../../")
+from tests import pyunit_utils
+import h2o
+import threading
+
+def h2ocluster_shutdown():
+    """
+    Python API test: h2o.cluster().shutdown(prompt=False)
+    """
+    thread = threading.Thread(target=call_shutdown)
+    thread.daemon =True
+
+    try:
+        thread.start()
+        thread.join(1.0)
```

Comment:
Basically, I call shut down in a thread that ask for user input and it will just wait there and do nothing until a user type in y.

Therefore, I started another thread to just wait for one second and kill the thread waiting for user input.  Since the user never types in y, the cluster is not shut down.  Hence, no bad effect is experienced, no cluster is shut down but we do get to test the shutdown command.